pkg/cloud/services/compute/instance.go

```diff
@@ -380,14 +380,6 @@ func (s *Service) createInstanceImpl(eventObject runtime.Object, openStackCluste
 	return createdInstance, nil
 }
 
-// getPortName appends a suffix to an instance name in order to try and get a unique name per port.
```

Comment:
Why did I move this you will inevitably ask?  Well, problem is I want to associate the test with the function that causes the issue in the first place.  Problem is I cannot just export this and include it because of circular dependencies.  That led me to going, well this should be a `networking_test` package anyway, which then led to to not being able to create the networking service due to the client filed not being exported, which then became ugly because of a hacky `NewWithClientForTestOnly()` function in the production code.  Moving it was by far the simplest solution.